Added new pack: Email Hippo (#31167)
demisto/content: Packs/EmailHippo/Integrations/EmailHippo/README.md

> This is the Email Hippo integration for verify email sources as fake emails that were used as part of phishing attacks.

Reviewer: ```suggestion
This is the Email Hippo integration used to verify email sources as fake emails that were used as part of phishing attacks.
```

.
This integration was integrated and tested with version 2.0.1551 of Email Hippo.

## Configure Email Hippo on Cortex XSOAR

1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
2. Search for Email Hippo.
3. Click **Add instance** to create and configure a new integration instance.

    | **Parameter** | **Description** | **Required** |
    | --- | --- | --- |

>     | MORE Server URL (e.g. https://api.hippoapi.com) |  | True |
>     | Email Hippo WHOIS Server URL (e.g. https://api.whoishippo.com) |  | True |

Reviewer: ```suggestion
    | MORE Server URL (e.g., https://api.hippoapi.com) |  | True |
    | Email Hippo WHOIS Server URL (e.g., https://api.whoishippo.com) |  | True |
```

    | MORE API Key |  | True |
    | WHOIS API Key |  | True |
    | Source Reliability | Reliability of the source providing the intelligence data. | False |

>     | Create relationships | Create relationships between indicators as part of Enrichment. | False |

Reviewer: ```suggestion
    | Create relationships | Create relationships between indicators as part of enrichment. | False |
```

    | Trust any certificate (not secure) |  | False |
    | Use system proxy settings |  | False |

4. Click **Test** to validate the URLs, token, and connection.

## Commands

You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
After you successfully execute a command, a DBot message appears in the War Room with the command details.

### email-hippo-email-quota-get

***

> Get the Email quota from the API.

Reviewer: ```suggestion
Get the email quota from the API.
```

#### Base Command

`email-hippo-email-quota-get`

#### Input

There are no input arguments for this command.

#### Context Output

| **Path** | **Type** | **Description** |
| --- | --- | --- |
| EmailHippo.Quota.quotaUsed | String | Total quota used. | 
| EmailHippo.Quota.quotaRemaining | String | The remaining quota. | 

#### Command example
```!email-hippo-email-quota-get```
#### Context Example
```json
{
    "EmailHippo": {
        "Quota": {
            "accountId": 7031,
            "errorSummary": "Valid",
            "nextQuotaResetDate": "2023-12-28T00:00:00",
            "quotaRemaining": 99,
            "quotaUsed": 1,
            "reportedDate": "2023-11-28T12:49:16.0260781Z"
        }
    }
}
```

#### Human Readable Output

>### Email quota
>|Email Quota remaining|Email Quota used|
>|---|---|
>| 99 | 1 |

### email

***

> Return Email information and reputation.

Reviewer: ```suggestion
Return email information and reputation.
```

#### Base Command

`email`

#### Input

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |

> | email | The email address to validate (CSV). | Required | 

Reviewer: ```suggestion
| email | A comma-separated list of email addresses to validate. | Required | 
```

                "blockLists": [
                    {
                        "blockListName": "spamhaus",
                        "isListed": false,
                        "listedMoreInfo": null,
                        "listedReason": null
                    }
                ],
                "domainRiskScore": 10,
                "formatRiskScore": 0,
                "isDarkWebEmailAddress": false,
                "isDisposableEmailAddress": true,
                "isGibberishDomain": false,
                "isGibberishUser": false,
                "overallRiskScore": 10,
                "profanityRiskScore": 0
            },
            "spamTrapAssess": {
                "isSpamTrap": false,
                "spamTrapDescriptor": null
            },
            "version": {
                "doc": "https://api-docs.emailhippo.com/en/latest/",
                "v": "More-(1.2.1091)"
            }
        }
    }
}
```

#### Human Readable Output

>### Email test@example.com
>|Hippo Trust Score|Inbox quality score|Result|Spam risk score|
>|---|---|---|---|
>| Low | DoNotSend | result: Unverifiable<br/>reason: DomainIsWellKnownDea | Block |

### domain

***

> Returns Domain information and reputation.

Reviewer: ```suggestion
Returns domain information and reputation.
```

#### Base Command

`domain`

#### Input

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |
| domain | The domain to query (CSV). | Required | 

#### Context Output

| **Path** | **Type** | **Description** |
| --- | --- | --- |
| DBotScore.Indicator | String | The indicator that was tested. | 
| DBotScore.Reliability | String | The reliability score of the vendor. | 
| DBotScore.Score | Number | The actual score. | 
| DBotScore.Type | String | The indicator type. | 
| DBotScore.Vendor | String | The vendor used to calculate the score. | 
| Domain.Name | String | The name of the domain that was checked. | 

> | Domain.NameServers | String | Name servers of the domain. | 

Reviewer: ```suggestion
| Domain.NameServers | String | Name of the servers of the domain. | 
```

| Domain.UpdatedDate | Date | The date that the domain was last updated. | 

> | Domain.CreationDate | Date | The creation date of the domain. Format is ISO8601 \(i.e. '2020-04-30T10:35:00.000Z'\). | 

Reviewer: ```suggestion
| Domain.CreationDate | Date | The creation date of the domain. Format is ISO8601 \(i.e.,'2020-04-30T10:35:00.000Z'\). | 
```

| Domain.Registrar.Name | String | The name of the registrar. | 
| Domain.Registrar.AbuseEmail | String | The email address of the contact for reporting abuse. | 

> | Domain.Registrar.AbusePhone | String | The phone number of contact for reporting abuse. | 

Reviewer: ```suggestion
| Domain.Registrar.AbusePhone | String | The phone number of the contact for reporting abuse. | 
```

| Domain.Admin.Country | String | The country of the domain administrator. | 
| Domain.Admin.Email | String | The email address of the domain administrator. | 
| Domain.Admin.Name | String | The name of the domain administrator. | 
| Domain.Admin.Phone | String | The phone number of the domain administrator. | 
| Domain.Tech.Country | String | The country of tech administrator. | 
| Domain.Tech.Name | String | The name of the tech administrator. | 

> | Domain.Tech.Email | String | The email of tech administrator. | 

Reviewer: ```suggestion
| Domain.Tech.Email | String | The email of the tech administrator. | 
```

> | Domain.Tech.Email | String | The email of tech administrator. | 

Reviewer: ```suggestion
| Domain.Tech.Email | String | The email of the tech administrator. | 
```

| Domain.Tech.Organization | String | The organization of the tech administrator. | 
| Domain.WHOIS.NameServers | String | A CSV string of name servers, for example 'ns1.bla.com, ns2.bla.com'. | 

> | Domain.WHOIS.CreationDate | Date | The creation date of the domain. Format is ISO8601 \(i.e. '2020-04-30T10:35:00.000Z'\). | 
> | Domain.WHOIS.UpdatedDate | Date | The date when the domain was last updated. Format is ISO8601 \(i.e. '2020-04-30T10:35:00.000Z'\). | 

Reviewer: ```suggestion
| Domain.WHOIS.CreationDate | Date | The creation date of the domain. Format is ISO8601 \(i.e., '2020-04-30T10:35:00.000Z'\). | 
| Domain.WHOIS.UpdatedDate | Date | The date when the domain was last updated. Format is ISO8601 \(i.e., '2020-04-30T10:35:00.000Z'\). | 
```
